2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_setup.py:_flush():71] setting env: {}
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_setup.py:_flush():71] setting login settings: {}
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_init.py:_log_setup():371] Logging user logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/MADDPG/wandb/run-20221113_150410-1usxpmu6/logs/debug.log
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_init.py:_log_setup():372] Logging internal logs to /home/josyula/Programs/MAS_Project/gym_envs_urdf/examples/MADDPG/wandb/run-20221113_150410-1usxpmu6/logs/debug-internal.log
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9f3d8a7580>
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_init.py:init():404] calling init triggers
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-11-13 15:04:10,938 INFO    MainThread:31114 [wandb_init.py:init():460] starting backend
2022-11-13 15:04:10,938 INFO    MainThread:31114 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-13 15:04:10,948 INFO    MainThread:31114 [backend.py:ensure_launched():216] starting backend process...
2022-11-13 15:04:10,952 INFO    MainThread:31114 [backend.py:ensure_launched():221] started backend process with pid: 31188
2022-11-13 15:04:10,953 INFO    MainThread:31114 [wandb_init.py:init():469] backend started and connected
2022-11-13 15:04:10,991 INFO    MainThread:31114 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-11-13 15:04:10,993 INFO    MainThread:31114 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-13 15:04:10,993 INFO    MainThread:31114 [wandb_init.py:init():533] updated telemetry
2022-11-13 15:04:11,005 INFO    MainThread:31114 [wandb_init.py:init():563] communicating current version
2022-11-13 15:04:11,836 INFO    MainThread:31114 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-13 15:04:11,837 INFO    MainThread:31114 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-11-13 15:04:11,977 INFO    MainThread:31114 [wandb_init.py:init():606] starting run threads in backend
2022-11-13 15:04:16,982 INFO    MainThread:31114 [wandb_run.py:_console_start():1810] atexit reg
2022-11-13 15:04:16,982 INFO    MainThread:31114 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-11-13 15:04:16,982 INFO    MainThread:31114 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-11-13 15:04:16,983 INFO    MainThread:31114 [wandb_run.py:_redirect():1745] Redirects installed.
2022-11-13 15:04:16,983 INFO    MainThread:31114 [wandb_init.py:init():633] run started, returning control to user process
2022-11-13 15:06:24,428 INFO    MainThread:31114 [wandb_init.py:_pause_backend():284] pausing backend
2022-11-13 15:06:24,428 INFO    MainThread:31114 [jupyter.py:save_ipynb():374] not saving jupyter notebook
